FirebaseAuthUI/FIRAuthPickerViewController.m

```diff
@@ -134,8 +134,12 @@ - (void)viewDidLayoutSubviews {
 #pragma mark - Actions
 
 - (void)signInWithEmail {
-  UIViewController *controller =
-      [[FIREmailEntryViewController alloc] initWithAuthUI:self.authUI];
+  UIViewController *controller;
+  if ([self.authUI.delegate respondsToSelector:@selector(emailEntryViewControllerForAuthUI:)]) {
+    controller = [self.authUI.delegate emailEntryViewControllerForAuthUI:self.authUI];
+  } else {
+    controller = [[FIREmailEntryViewController alloc] initWithAuthUI:self];
+  }
```

Comment:
We already have same situation with authViewController. I guess this check is redundant (different code style over project)